vendor/github.com/prometheus/prometheus/model/histogram/float_histogram.go

```diff
@@ -177,6 +177,21 @@ func (h *FloatHistogram) Scale(factor float64) *FloatHistogram {
 	return h
 }
 
+// Div works like Scale but divides instead of multiplies.
```

Comment:
[not blocking] Just noticed this comment refers to the old method name:
```suggestion
// Div works like Mul but divides instead of multiplies.
```

Reply:
Thanks, good catch! Will fix upstream.